staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -247,6 +254,13 @@ NextTest:
 }
 
 func TestServerRunWithSNI(t *testing.T) {
+	generate := os.Getenv("GENERATE_TEST_CERTS") == "true"
```

Comment:
I would also be fine with a `createIfNotExists` mechanism. One magic env var less.